docs/UserGuide.md

```diff
@@ -377,10 +380,10 @@ Examples:
 
 Shows the most recent 10 commands that was inputted.
 Allows cycling through those commands to reduce the need of retyping similar commands fully.
+The history is looped, meaning that when you reach the end, the next cycle will bring it back to the start.
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:** Every new valid command will reset the index back to the top of the history. 
+**Tip:** Every new valid command will reset the index back to the top of the history.
 Click Down Arrow to get the most recent command immediately and Up Arrow to get the last command in the history.
-</div>
```

Comment:
Actually why are we removing the :bulb: div? Is it not fine for it to remain?

Reply:
Oops i forgot to edit back because initially wanted to put the loop part together but decided against it